frontend/src/pages/ErrorsV2/ErrorStackTrace/ErrorStackTrace.module.css

```diff
@@ -57,13 +57,15 @@
 .collapsibleWrapper {
 	width: 100%;
 
-	&#{&} {
-		h2 {
-			font-size: 14px !important;
-			font-weight: 400;
-		}
+	/* Not quite sure what the original selector here was doing, replacing with simple nesting for now? */
+	/* &#{&} { */
```

Comment:
As mentioned in the comment, my SCSS-fu is pretty rusty and I couldn't figure out what this selector was doing, so this was my probably incorrect attempt at replicating it. 

But on a closer look, I also couldn't find any references to `collapsibleWrapper` anywhere, so maybe this entire class is safe to remove?